packages/camera/camera_web/lib/src/camera_settings.dart

```diff
@@ -8,8 +8,10 @@ import 'dart:ui';
 import 'package:camera_platform_interface/camera_platform_interface.dart';
 import 'package:camera_web/src/types/types.dart';
 import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
 
-/// A utility to fetch and map camera settings.
+/// A utility to fetch, map camera settings and
+/// obtain the camera stream.
 class CameraSettings {
```

Comment:
Is it time to find a better name for `CameraSettings`? Is it maybe doing too much? Slightly confusing naming with `CameraOptions`?

Maybe the parts regarding the management of the Camera Stream should be moved to a new class? Is this class more of a `CameraStreamsHelper`/`Handler`?

Reply:
Good point! It is currently responsible for getting the camera stream based on `CameraOptions`, getting the facing mode setting based on the camera video track and mapping camera video track settings. How about renaming it to a `CameraService` (this way we will no longer mix settings with options)? 

I'm also okay with extracting the camera stream logic to a separate class (`CameraStreams.getStreamForOptions` maybe?) if you prefer it that way.

Reply:
`CameraService` sounds good to me. For the conversion functions, what I've been doing is to have a fat "convert.dart" with a bunch of functions there that translate from one type to my own (and vice-versa). See [convert.dart in google maps](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart). I normally prefer those functions to be stateless, so they can be just utility functions (they don't need to belong to any class or namespace).